.github/workflows/tests.yml

```diff
@@ -25,16 +49,26 @@ jobs:
         git checkout --force -B master FETCH_HEAD
         cd -
 
+        if [ "$RUNNER_OS" = "Linux" ]; then
+          sudo chown -R "$USER" "$HOMEBREW_PREFIX"
+        fi
+
+    - name: Set up Ruby
+      if: matrix.os == 'ubuntu-latest'
+      uses: actions/setup-ruby@master
+      with:
+        ruby-version: '2.6'
+
     - name: Install taps
       run: |
         brew tap homebrew/test-bot
-        brew update-reset Library/Taps/homebrew/homebrew-test-bot
+        brew update-reset "$(brew --repo)/Library/Taps/homebrew/homebrew-test-bot"
 
     - name: Cache Bundler RubyGems
       id: cache
       uses: actions/cache@v1
       with:
-        path: /usr/local/Homebrew/Library/Homebrew/vendor/bundle/
+        path: ${{ steps.set-up-homebrew.outputs.brew-repo }}/Library/Homebrew/vendor/bundle/
```

Comment:
Does it not make sense to make these consistent? Is there a reason to do `brew --repo` in one and `outputs.brew-repo` in another?

Reply:
We can't use `brew --repo` here and it's a little easier to follow than the Actions version.

Reply:
@Bo98 as-of latest version it's handled slightly differently anyway

Reply:
That makes sense. :+1: